src/sentry/api/endpoints/data_scrubbing_selector_suggestions.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+from django.utils.translation import ugettext
+
+from sentry import eventstore
+from sentry.api.base import DocSection
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.utils.apidocs import scenario, attach_scenarios
+
+from sentry_relay import pii_selectors_from_event
+
+DEFAULT_SUGGESTIONS = [
+    {"type": "value", "value": "**", "description": ugettext("everywhere")},
+    {"type": "value", "value": "password", "description": ugettext('attributes named "password"')},
+    {"type": "value", "value": "$error.value", "description": ugettext("the exception value")},
+    {"type": "value", "value": "$message", "description": ugettext("the log message")},
+    {
+        "type": "value",
+        "value": "extra.'My Value'",
+        "description": ugettext('the key "My Value" in "Additional Data"'),
+    },
+    {
+        "type": "value",
+        "value": "extra.**",
+        "description": ugettext('everything in "Additional Data"'),
+    },
+    {
+        "type": "value",
+        "value": "$http.headers.x-custom-token",
+        "description": ugettext("the X-Custom-Token HTTP header"),
+    },
+    {"type": "value", "value": "$user.ip_address", "description": ugettext("the user IP address")},
+    {
+        "type": "value",
+        "value": "$frame.vars.foo",
+        "description": ugettext('the local variable "foo"'),
+    },
+    {
+        "type": "value",
+        "value": "contexts.device.timezone",
+        "description": ugettext("The timezone in the device context"),
```

Comment:
```suggestion
        "description": ugettext("the timezone in the device context"),
```
The other the words are lower cased.